notebooks/105-language-quantize-bert/105-language-quantize-bert.ipynb

```diff
@@ -7,8 +7,8 @@
     "id": "rQc-wXjqrEuR"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    for input_layer in model.inputs:
</pre><p>Sorry, I know that it is not your code, but could you please fix?</p><p> </p><p>instead of reshaping model in cycle for each input layer here should be:</p><p> </p><p> </p><p><code>dynamic_shapes = {}</code></p><p><code>for input_layer in model.inputs:</code></p><p><code>&nbsp;&nbsp;&nbsp;input_shape = input_layer.partial_shape</code></p><p><code>&nbsp;&nbsp;&nbsp;input_shape[1] = -1</code></p><p><code>dynamic_shapes[input_layer] = input_shape</code></p><p><code>model.reshape(dynamic_shapes)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/941/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>